packages/core/src/components/toast/_toast.scss

```diff
@@ -47,7 +47,7 @@ $toast-intent-colors: (
 
     &:focus {
       // blue outline color shows poorly on colored bg
-      outline-color: rgba($white, 0.5);
+      outline-color: rgba($text-color, 0.75);
```

Comment:
Was this intentional? Seems odd to use the text color for the outline.

Reply:
I left an exploration of this here: https://github.com/palantir/blueprint/pull/6854#issuecomment-2201197249

The text color is what allows us to easily flip to a dark outline when rendering against a light color background. Alternate option to define a separate focus color variable, but I'm not sure what we'd really choose differently - these are already against intent backgrounds so it's a bit of a weird case.

Reply:
I'm not saying the color itself is weird, and choosing the same color seems fine. It's just odd to use the variable for _text_ color for something that isn't text.

Reply:
ah yes - updated to `$text-and-focus-color`, didn't think of anything good to combine them generically, ex `$content-color` doesn't sound right